stories/Cookiebar/Cookiebar.stories.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withA11y } from '@storybook/addon-a11y'
+import { withInfo } from '@storybook/addon-info'
+
+
+import Esempi from './docs/example.md'
+
+import { CookieBar,CookieBarButtons,CookieBarButton } from '../../src'
+
+class CookieBarComponent extends React.Component {
+    defaultState = {
+        clicked : true
+    }
+    state = {
+        ...this.defaultState,
+        clicked: false
+    }
+    toggle = () => {
+        this.setState({
+            ...this.defaultState,
+            clicked: !this.state.clicked
+        })
+    }
+
+    render(){
+        if(this.state.clicked==false){
```

Comment:
you can just return if the cookiebar is closed
```js
if( !open ){
  return;
}
... open branch
```

Reply:
@dej611 I added a knob as you suggested. If I state is controlled by knob and not on user onclick, should i remove the toggle() and onclick?? 
Also when `return;` happens , we get an error saying that nothing is to be returned by render. 
Please suggest how to proceed with this?
```
const CookieBarComponent = () => {
    const open = boolean('isOpen',true)
    function toggle(){
    
    }
    if(!open) return;
    else{
        return (
            <CookieBar> 
                <p>Questo sito utilizza cookie tecnici, analytics e di terze parti. <br/>Proseguendo nella navigazione accetti l’utilizzo dei cookie.</p> 
                <CookieBarButtons>  
                    <CookieBarButton href="#">Preferenze<span class="sr-only">cookies</span></CookieBarButton>
                    <CookieBarButton onClick={()=>this.toggle()}>Accetto<span class="sr-only"> i cookies</span></CookieBarButton> 
                </CookieBarButtons>
            </CookieBar>
        )
    }
}
```

Reply:
@dej611 `return;` would show an error since null cannot be rendered.

Reply:
Not true, `undefined` is managed by JSX not rendering anything.

Reply:
@dej611 I tried this out, and it did show me an error saying that null cannot be rendered.

Reply:
Just return `null` rather than just `return` and it will work for sure:

```jsx
if( !open ){
  return null;
}
... open branch
```

Reply:
BTW you are right, `undefined` is not supported as direct JSX element, but only within a valid JSX element `<div>{undefined}</div>`.
The problem is that also your original code is (indirectly) returning undefined when the `clicked` state was true.